### DISCUSSION TITLE
Hi, what does change the last update?

### DISCUSSION THREAD

**Discussion by HumanistAtypik** [YU000jp/logseq-plugin-short-namespaces#16] [Q&A]
I'm not a good English speaker, sorry

**Comment by YU000jp** [comment]
https://discord.com/channels/725182569297215569/1116515366508711996/1132674511465091156
I had intended to apply shortened displays to links within Linked References and Queries, but it inadvertently led to a bug. I thought I had noticed and released it, but it seems like it wasn't actually issued. Consequently, we released a patch to apply the rebate.
